index.js

```diff
@@ -2,17 +2,24 @@ const { codeFrameColumns } = require("@babel/code-frame");
 const Worker = require("jest-worker").default;
 const serialize = require("serialize-javascript");
 
+function reject(raw, rejected) {
+  return Object.keys(raw)
+    .filter(key => !rejected.includes(key))
+    .reduce((obj, key) => {
+      obj[key] = raw[key];
+      return obj;
+    }, {});
+}
+
 function uglify(userOptions = {}) {
   if (userOptions.sourceMap != null) {
     throw Error("sourceMap option is removed, use sourcemap instead");
   }
 
   const minifierOptions = serialize(
-    Object.assign({}, userOptions, {
-      sourceMap: userOptions.sourcemap !== false,
-      sourcemap: undefined,
-      numWorkers: undefined
-    })
+    reject(Object.assign({}, userOptions, {
+      sourceMap: userOptions.sourcemap !== false
+    }), [ "sourcemap" ])
```

Comment:
I would prefer to use `delete` operator.

Reply:
the devDependence of serialize-javascript update to 1.9.0, leads to uglify-js always fail,
the message is Error: `sourcemap` is not a supported option

Reply:
the Dependence of serialize-javascript update to 1.9.0, leads to uglify-js always fail,
the message is Error: "sourcemap is not a supported option"

Reply:
https://github.com/TrySound/rollup-plugin-terser/blob/master/index.js#L38

Reply:
@TrySound updated